### PR TITLE
Dataset panel user documentation

### PR DESCRIPTION
**Pull request by matthewma7**
Add a user documentation for Dataset panel. Also removed some out of date documentation and applied some documentation structure change.

### CODE REVIEW

File: docs/datasets-panel.rst [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+Dataset actions
+===============
+
+Visualize dataset
+-----------------
+.. image:: img/datasets_panel-globe_button.png
+
+Click the globe button after the dataset name to add the dataset to layer panel and visualize the dataset.
+
+Once the dataset is visualized. The globe button will be replaced by a remove button. Click this button to remove the dataset from visualization.
```
Comment (dorukozturk) [inline docs/datasets-panel.rst]: Once the dataset is visualized, the globe button ...

File: docs/datasets-panel.rst [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+|
+
+Dataset actions
+===============
+
+Visualize dataset
+-----------------
+.. image:: img/datasets_panel-globe_button.png
+
+Click the globe button after the dataset name to add the dataset to layer panel and visualize the dataset.
```
Comment (dorukozturk) [inline docs/datasets-panel.rst]: I think we don't have to say click on the button. We can rather use something like "the globe button adds the dataset to the layer panel and plots the dataset."

File: docs/datasets-panel.rst [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+.. image:: img/datasets_panel-dataset_shown_remove.png
+
+|
+
+Delete dataset
+--------------
+.. image:: img/datasets_panel-trash_button.png
+
+Click the trash can button after dataset name to delete one dataset. A confirmation prompt will be shown before deleting the dataset. Dataset deletion is irreversible.
```
Comment (dorukozturk) [inline docs/datasets-panel.rst]: Same here. We can use something like: "trash button deletes a dataset"

File: docs/datasets-panel.rst [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+|
+
+Dataset info
+------------
+.. image:: img/datasets_panel-info_button.png
+
+Click this button will show a popup window displaying dataset related metadata
```
Comment (dorukozturk) [inline docs/datasets-panel.rst]: Same here. "Info button shows a popup window which displays dataset related metadata"

File: docs/datasets-panel.rst [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+|
+|
+
+Dataset selection actions
+=========================
+There are features available only when there are datasets in selected states. To select a dataset or group of datasets, hovering cursor before a dataset or group name, then check the checkbox.
```
Comment (dorukozturk) [inline docs/datasets-panel.rst]: "hover the cursor on a dataset or category icon, then check the checkbox"

File: docs/datasets-panel.rst [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+Dataset selection action will appear at the top right corner of the panel. Some are only available when one dataset is selected. Some are available when multiple datasets are selected.
+
+|
+
+Visualize datasets boundaries
+-----------------------------
+.. image:: img/datasets_panel-selection-boundary_button.png
+
+This action will visualize boundary bounding box of one or more selected datasets.
```
Comment (dorukozturk) [inline docs/datasets-panel.rst]: "visualize bounding box"

File: docs/datasets-panel.rst [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+
+When boundaries are being visualized, two additional actions will be available to toggle the label of boundaries or remove all boundaries.
+
+|
+
+Filter dataset list boundary
+----------------------------
+.. image:: img/datasets_panel-selection-filter_button.png
+
+When there is one dataset selected, click on this action will filter the list of datasets available and only show datasets that overlap the boundary of selected dataset.
```
Comment (dorukozturk) [inline docs/datasets-panel.rst]: "clicking on this action"

File: docs/datasets-panel.rst [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+Filter dataset list boundary
+----------------------------
+.. image:: img/datasets_panel-selection-filter_button.png
+
+When there is one dataset selected, click on this action will filter the list of datasets available and only show datasets that overlap the boundary of selected dataset. 
+
+.. image:: img/datasets_panel-filtered_remove-filter-button.png
+
+When the list of datasets being filtered a remove action will be available to stop the filtering.
```
Comment (dorukozturk) [inline docs/datasets-panel.rst]: "are being filtered, a remove action ..."

File: docs/datasets-panel.rst [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+
+|
+
+Visualize or delete multiple datasets
+-------------------------------------
+.. image:: img/datasets_panel-selection_multiple-globe-remove-buttons.png
+
+When more than one dataset has been selected, a globe and trash can icon will appear on the top right as selection action. They will visualize or delete all selected datasets.
```
Comment (dorukozturk) [inline docs/datasets-panel.rst]: "top right as selection actions".
"Those buttons are used for plotting or deleting all selected datasets."

**Comment by matthewma7** [comment]
@dorukozturk Thank you for your feedback. They are really helpful. I have made updates accordingly.

File: docs/datasets-panel.rst [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+==============
+Dataset panel is the central place to manage your datasets. Datasets are available to all sessions of a user. A user could choose to share dataset with other users.
+
+
+Upload
+======
+.. image:: img/datasets_panel-upload_button.png
+
+This is the initial state of dataset panel.
+You could use this upload button to upload dataset. See :doc:`/getting-started` for more detail.
```
Comment (dorukozturk) [inline docs/datasets-panel.rst]: You could use --> A user could 
Just like the one on line 4.

File: docs/datasets-panel.rst [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+Dataset actions
+===============
+
+Visualize dataset
+-----------------
+.. image:: img/datasets_panel-globe_button.png
+
+The globe button after the dataset name adds the dataset to layer panel and visualize the dataset.
+
+Once the dataset is visualized, the globe button will be replaced by a remove button. Click this button to remove the dataset from visualization.
```
Comment (dorukozturk) [inline docs/datasets-panel.rst]: Click this button to remove the dataset from visualization --> Click on this button to remove the dataset from the map

File: docs/datasets-panel.rst [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+
+The trash button after dataset name deletes one dataset. A confirmation prompt will be shown before deleting the dataset. Dataset deletion is irreversible.
+
+|
+
+Dataset info
+------------
+.. image:: img/datasets_panel-info_button.png
+
+Info button shows a popup window displaying dataset related metadata
```
Comment (dorukozturk) [inline docs/datasets-panel.rst]: Add a . at the end

File: docs/datasets-panel.rst [hunk 13]
```diff
@@ -0,0 +1,104 @@
...
+
+Dataset selection action will appear at the top right corner of the panel. Some are only available when one dataset is selected. Some are available when multiple datasets are selected.
+
+|
+
+Visualize datasets boundaries
+-----------------------------
+.. image:: img/datasets_panel-selection-boundary_button.png
+
+This button visualizes boundary bounding box of one or more selected datasets.
```
Comment (dorukozturk) [inline docs/datasets-panel.rst]: boundary bounding box --> bounding box or boundary

**Comment by matthewma7** [comment]
@dorukozturk Thank you for the review. 👍

**Pull request merged**